presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -87,110 +81,219 @@ fun EventScreen(
+                                )
                             }
+                        )
+                    }
+
+                    if(eventScreenState.selectedVisitorFilterType == VisitorFilterType.ALL || eventScreenState.selectedVisitorFilterType == VisitorFilterType.GOING) {
+                        /** Going section */
+                        item {
+                            Text(
+                                text = stringResource(id = R.string.going),
+                                fontWeight = FontWeight.Medium,
+                                fontSize = 16.sp,
+                                color = MaterialTheme.colorScheme.visitorTextFontColor
+                            )
+                            Spacer(modifier = Modifier.height(16.dp))
+
                         }
-                    )
-                    Spacer(modifier = modifier.height(26.dp))
+                        if(eventScreenState.filteredVisitorsGoing.isNotEmpty()) {
+                            items(eventScreenState.filteredVisitorsGoing) { attendee ->
+                                VisitorItem(
+                                    initials = attendee.fullName.toInitials(),
+                                    userId = attendee.userId,
+                                    fullName = attendee.fullName,
+                                    onDeleteClicked = { userId ->
+                                        eventScreenEvent(EventScreenEvent.OnDeleteVisitor(userId))
+                                    },
+                                    isCreator = eventScreenState.host == attendee.userId
+                                )
+                            }
+                        }
+                    }
 
-                    VisitorFilter(
-                        modifier = Modifier.fillMaxWidth(),
-                        selectedVisitorType = VisitorFilterType.ALL,
-                        onSelectedTypeClicked = {},
-                        onAddVisitorClicked = {
-                            eventScreenEvent(EventScreenEvent.OnShowVisitorDialog(shouldShowVisitorDialog = true))
+                    /** Not going section */
+                    if(eventScreenState.selectedVisitorFilterType == VisitorFilterType.ALL || eventScreenState.selectedVisitorFilterType == VisitorFilterType.NOT_GOING) {
+                        item {
+                            Text(
+                                text = stringResource(id = R.string.not_going),
+                                fontWeight = FontWeight.Medium,
+                                fontSize = 16.sp,
+                                color = MaterialTheme.colorScheme.visitorTextFontColor
+                            )
+                            Spacer(modifier = Modifier.height(16.dp))
                         }
-                    )
 
-                    Spacer(modifier = modifier.height(26.dp))
+                        if(eventScreenState.filteredVisitorsNotGoing.isNotEmpty()) {
```

Comment:
Applies the same comments as mentioned in the "Going" section

Reply:
@MKiperszmid Updated